vector/src/main/java/fr/gouv/tchap/activity/TchapLoginActivity.java

```diff
@@ -676,7 +677,7 @@ void onForgotPasswordClick() {
         onClick();
 
         // parameters
-        final String email = mForgotEmailTextView.getText().toString().trim();
+        final String email = mForgotEmailTextView.getText().toString().toLowerCase(VectorLocale.INSTANCE.getApplicationLocale()).trim();
```

Comment:
I think it's ok to use VectorLocale.INSTANCE.getApplicationLocale(), but for e-mails, is it really better that using default locale?

Reply:
you're right I will remove the locale here

Reply:
After thinking I'm wondering whether we should not use `Locale.ROOT`. wdyt?